src/vhtml.js

```diff
@@ -36,7 +36,7 @@ export default function h(name, attrs) {
 
 		while (stack.length) {
 			let child = stack.pop();
-			if (child) {
+			if (child !== undefined && child !== null) {
```

Comment:
This would allow `<div>{true}</div>` to render `<div>true</div>`. Do we want that? The VDOM libs ignore booleans.

In any case, this statement can compress to:

```suggestion
			if (child != null) {
```

Reply:
It also allowed it before so we could leave as is or correct it, up to you. The shorter version while shorter is not explicit, it's better to always use the `===` and `!==` operators. Just my thoughts.

Reply:
I would concur with @dburles that being more explicit is often clearer. Also on the `<div>{true}</div>` debate mimicking the behaviour of React/Preact etc. seems intuitive.